data.md

```diff
@@ -303,7 +323,7 @@ All data is also available on [our GitHub repository for analysis](https://githu
     left: 0;
     right: 0;
     height: 100px;
-    background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 50%, rgba(255, 255, 255, 1) 100%);
+    background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 80%, rgba(255, 255, 255, 1) 100%);
```

Comment:
Since we now clarify that it is a "preview" table, we can make the table content slightly more clearly visible.